changelog.txt

```diff
@@ -7,6 +7,7 @@
 * Fix - Fixed missing file error for removed CSS file.
 * Add - Currency deletion confirmation modal for currencies that are bound to an UPE method.
 * Fix - Currency switcher does not affect order confirmation screen prices.
+* Fix - Refund button is missing from orders that saved with errors.
```

Comment:
Will it be better to rephrase it to describe what is done than what the error was? I mean something like 'Fill missing order_intent_info even if an exception occurs`?

Reply:
Yep, sounds good!
Fixed in https://github.com/Automattic/woocommerce-payments/pull/2936/commits/f567b0db0cbbf0bc4a656fd7645ff165a89ee88a and https://github.com/Automattic/woocommerce-payments/pull/2936/commits/c992f1fa3169e9b4beffc0e21acd67c71701a35c.